spec/system/pupils/public_displays_spec.rb

```diff
@@ -0,0 +1,202 @@
+require 'rails_helper'
+
+describe 'Pupil dashboard' do
```

Comment:
Should this be "pupil dashboard public displays" or similar rather than "Pupil dashboard"?